### DISCUSSION TITLE
Showing invalid subdomains ?

### DISCUSSION THREAD

**Discussion by Anthonymcqueen21** [projectdiscovery/subfinder#714] [Q&A] [answered]
Hello my problem is why does subfinder search for invalid and or outdated subdomains ?

**Comment by vzamanillo** [comment, marked as answer]
Subfinder does not store any information about subdomains, it queries third party services to show the results, but, you can pipeline the results to other tools like httpx to probe if they are still active.

**Reply by Anthonymcqueen21** [reply]
What do you mean i am still learning this tool all i get is invalid subdomains that are either long gone or just dont exist anymore and lead me to nowhere.

**Comment by ehsandeep** [comment]
@Anthonymcqueen21 subfinder is passive subdomain enumeration tool that pulls out all the known historical subdomains from various sources that could be valid or invalid, if you are interested in valid subdomains only, you can use `-nW` option to filter inactive subdomains.

**Reply by Anthonymcqueen21** [reply]
I have used the -nW option and it does not show valid subdomains, why is that ? ... How would a command of that nature look like when i want to do it ?

**Reply by vzamanillo** [reply]
Give us some practical example to try to reproduce this, meanwhile you can pipe the results to any other tool like [httpx](https://github.com/projectdiscovery/httpx) to probe if the domain is still active.

**Reply by Anthonymcqueen21** [reply]
Okay thank you.
